src/client/cli/commands/netAddress.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+const RestClient = require('../../api/RestClient')
+
+module.exports = {
+  command: 'netAddress',
+  describe: `Print the local node's network address in multiaddr format.`,
```

Comment:
plural: addresses!
Maybe also disambiguate as "routable addresses" (because it filters localhost/linklocal), but that's perhaps too much detail.